modules/tinymce/src/themes/silver/test/ts/browser/editor/ShowHideTest.ts

```diff
@@ -0,0 +1,105 @@
+import { Assertions, Chain, Guard, Pipeline, Log, UiFinder } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Editor as McEditor, ApiChains, UiChains } from '@ephox/mcagar';
+import { SugarBody, Visibility } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+UnitTest.asynctest('browser.tinymce/themes.silver.ShowHideTest', (success, failure) => {
+  Theme();
+
+  const base_url = '/project/tinymce/js/tinymce';
+
+  const cWaitForVisible = UiChains.cWaitForState(Visibility.isVisible);
+  const cWaitForInvisible = (label: string, selector: string) => Chain.control(
+    Chain.fromIsolatedChains([
+      Chain.injectThunked(SugarBody.body),
+      Chain.exists([
+        UiFinder.cNotExists(selector),
+        Chain.fromIsolatedChains([
+          UiFinder.cFindIn(selector),
+          Chain.mapper(Visibility.isVisible),
+          Assertions.cAssertEq(label, false)
+        ])
+      ])
+    ]),
+    Guard.tryUntil(label, 10, 10000)
```

Comment:
That's a very large timeout. Does it need to be that large? Also, could we just use the defaults here instead, which are 10 and 3000 IIRC?